src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+-------------------------------------------------------------
+-- Drop each of the schemas used by SDT and all their contents
+--------------------------------------------------------------
+--DROP SCHEMA IF EXISTS "SDT_USER" CASCADE;
+--DROP SCHEMA IF EXISTS "SDT_BATCH_USER" CASCADE;
+--DROP SCHEMA IF EXISTS "SDT_OWNER" CASCADE;
+
+-------------------------------------------------------------
+-- Drop each of the users used by SD
+--------------------------------------------------------------
+---- non schema owners first
+--DROP USER IF EXISTS "SDT_USER";
+--DROP USER IF EXISTS "SDT_BATCH_USER";
+---- now schema owners
+--DROP USER IF EXISTS "SDT_OWNER";
+
```

Comment:
Any reason why these lines have been commented out?  The comment on line 9 seems to be missing the "T" from the end of SDT.

Reply:
Because when run from Flyway, it's the first script and will be a new database. So only while continually re-testing, this is needed.